include/TabWidget.h

```diff
@@ -68,6 +69,8 @@ class TabWidget : public QWidget
 	void setTabTitleText( const QColor & c );
 	QColor tabSelected() const;
 	void setTabSelected( const QColor & c );
+	QColor tabTextSelected() const;
+	void setTabTextSelected( const QColor & c );
```

Comment:
```suggestion
	void setTabTextSelected(const QColor & c);
```

Reply:
Spaces fixed with commit 0fa18d405e7 where I have fixed style issues for all of `TabWidget`.